snap/snapcraft.yaml

```diff
@@ -10,6 +10,9 @@ platforms:
   amd64:
     build-on: [amd64]
     build-for: [amd64]
+  arm64:
+    build-on: [arm64]
+    build-for: [arm64]
```

Comment:
(optional to consider) According to https://snapcraft.io/docs/reference-architectures#core24 , if the key above is a standard architecture name, and build-on and build-ofr is the same, then they can be omitted:

```suggestion
```